driver-scala/build.gradle

```diff
@@ -29,22 +29,21 @@ dependencies {
     testRuntimeOnly project(path: ':driver-core', configuration: 'consumableTestRuntimeOnly')
 }
 
-
-configurations {
-    integrationTestCompile.extendsFrom testImplementation
-    integrationTestRuntime.extendsFrom testRuntime
-}
-
 sourceSets {
     integrationTest {
         scala.srcDir file('src/it/scala')
         scala {
-            compileClasspath += main.output + test.output
-            runtimeClasspath += main.output + test.output
+            compileClasspath += main.output
+            runtimeClasspath += main.output
         }
     }
 }
 
+configurations {
```

Comment:
Curious why order matters here.  It has to go below sourceSets?

Reply:
Order not important

Reply:
Moved this back in the config and fixed the intellij issue